corehq/apps/repeaters/models.py

```diff
@@ -50,7 +50,7 @@
 def simple_post_with_cached_timeout(data, url, expiry=60 * 60, force_send=False, *args, **kwargs):
     # no control characters (e.g. '/') in keys
     key = hashlib.md5(
-        '{0} timeout {1}'.format(__name__, url)
+        u'{0} timeout {1} {2}'.format(__name__, url, data).encode('utf-8')
```

Comment:
this is saying cache the error when sending the same data to the same url—that basically means we're deciding not cache to the fact that an endpoint is timing out, right? What we were doing before was definitely too broad, but just checking that we're making that decision.